arches/app/src/components/ControlledListManager/AddLabel.vue

```diff
@@ -0,0 +1,78 @@
+<script setup lang="ts">
+import arches from "arches";
+import { computed, ref } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import EditLabel from "@/components/ControlledListManager/EditLabel.vue";
+
+import type {
+    ControlledListItem,
+    Label,
+    LanguageMap,
+    ValueType,
+} from "@/types/ControlledListManager.d";
+
+const props: {
+    item: ControlledListItem;
+    languageMap: LanguageMap;
+    type: ValueType;
+    onInsert: (label: Label) => Promise<Label>;
+} = defineProps(["item", "languageMap", "type", "onInsert"]);
+
+const visible = ref(false);
```

Comment:
no ref typing?

Reply:
IIUC, [most ref types can be inferred by the compiler](https://vuejs.org/guide/typescript/composition-api.html#typing-ref).